test/tokens.js

```diff
@@ -39,24 +39,24 @@ describe("token operations", function() {
   it("can establish tokens with maximum", async ()=> {
     let {wallet, walletKey, chainId} = await createWalletWithChain();
 
-    resp = await wallet.establishCoin(chainId, walletKey, "token-a", 500);
+    resp = await wallet.establishToken(chainId, walletKey, "token-a", 500);
     assert.notEqual(resp.tip, null);
 
-    resp = await wallet.mintCoin(chainId, walletKey, "token-a", 250);
+    resp = await wallet.mintToken(chainId, walletKey, "token-a", 250);
     assert.notEqual(resp.tip, null);
 
-    resp = await wallet.mintCoin(chainId, walletKey, "token-a", 200);
+    resp = await wallet.mintToken(chainId, walletKey, "token-a", 200);
     assert.notEqual(resp.tip, null);
 
     try {
-      await wallet.mintCoin(chainId, walletKey, "token-a", 100);
-      assert.fail("Should not be able to mint more tokens than establishCoin maximum")
+      await wallet.mintToken(chainId, walletKey, "token-a", 100);
+      assert.fail("Should not be able to mint more tokens than establishToken maximum")
     } catch (err) {
       assert.notEqual(err, null);
     }
 
     // FIXME: Should pass, fix incoming in tupelo
-    // resp = await wallet.mintCoin(chainId, walletKey, "token-a", 50);
+    // resp = await wallet.mintToken(chainId, walletKey, "token-a", 50);
```

Comment:
This could be uncommented now, couldn't it? (Perhaps not in this PR, though.)

Reply:
yes, good call

Reply:
Actually looks like thats dependent on your draft PR for tokens - seems the PR i had done to fix this didn't get carried over into the tupelo-go-client.

Reply:
Oh, right, b/c the mint stuff wasn't defined all the way out to the RPC interface yet. 👍